src/state/CallViewModel/localMember/LocalTransport.ts

```diff
@@ -49,6 +46,8 @@ interface Props {
  *
  * @prop useOldestMember Whether to use the same transport as the oldest member.
  * This will only update once the first oldest member appears. Will not recompute if the oldest member leaves.
+ *
+ * @throws MatrixRTCTransportMissingError | FailToGetOpenIdToken
```

Comment:
👍 I wonder if we can have a linter rule that uses that information to ensure the caller handle the errors

Reply:
I searched a bit. not possible.